src/Repository/FileType.hpp

```diff
@@ -14,5 +14,6 @@ enum class FileType : uint8_t {
   FLARMNET,
   IGC,
   RASP,
-  XCI
+  XCI,
+  TASK
```

Comment:
Trailing comma!

Reply:
To extend the enum I need to add that comma. I believe this to be correct.

Reply:
Of course, but I mean you should add a trailing comma! The last guy (who added `XCI`) forgot it and now you had to modify an unrelated line.

Reply:
Got it. I split the commit in 2, one to fix the missing comma, and the second one to add the task incl. trailing comma.